src/map/pc.cpp

```diff
@@ -7363,6 +7363,9 @@ static void pc_checkallowskill(map_session_data *sd)
 		}
 
 		if (flag[SCF_REQUIRESHIELD]) { // Skills requiring a shield
+			// Only crusader-type classes require a shield for Defender
+			if (status == SC_DEFENDER && (sd->class_&MAPID_UPPERMASK) != MAPID_CRUSADER)
+				continue;
```

Comment:
It's a bit off topic, but what about other status like autoguard?

Reply:
Auto Guard, Reflect Shield and Endure wouldn't transfer at all through Devotion. They are currently coded incorrectly.